test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateTFMs.cs

```diff
@@ -35,10 +75,8 @@ public void Migrating_netcoreapp_project_Populates_TargetFrameworkIdentifier_and
             var testInputs = new MigrationRuleInputs(new[] { projectContext }, mockProj, mockProj.AddItemGroup(), mockProj.AddPropertyGroup());
             new MigrateTFMRule().Apply(testSettings, testInputs);
 
-            mockProj.Properties.Count(p => p.Name == "TargetFrameworkIdentifier").Should().Be(1);
-            mockProj.Properties.Count(p => p.Name == "TargetFrameworkVersion").Should().Be(1);
-            mockProj.Properties.First(p => p.Name == "TargetFrameworkIdentifier").Value.Should().Be(".NETCoreApp");
-            mockProj.Properties.First(p => p.Name == "TargetFrameworkVersion").Value.Should().Be("v1.0");
+            mockProj.Properties.Count(p => p.Name == "TargetFrameworks").Should().Be(1);
+            mockProj.Properties.First(p => p.Name == "TargetFrameworks").Value.Should().Be("netcoreapp1.0");
```

Comment:
Sounds good, I realized this is probably the reason for build failures here as well, the migration tests test that migrated projects build and run successfully.